src/FirebaseArduino.h

```diff
@@ -24,18 +24,97 @@
 #define FIREBASE_JSONBUFFER_SIZE 200
 #endif // FIREBASE_JSONBUFFER_SIZE
 
+/**
+ * Gateway class for Arduino clients to interact with the Firebase backend.
+ * This implementation is designed to follow Arduino best practices and favor
+ * simplicity over all else. 
+ * For more complicated usecases and more control see the Firebase class in 
+ * Firebase.h.
+ */
 class FirebaseArduino {
  public:
+  /**
+   * Must be called first. This sets the class up for use.
+   * \param host Your firebase db domain name, usually X.firebaseIO.com.
```

Comment:
done.
We can verify that and update the docs later.
Hostname is uppercase in the web UI, I don't have any preference though.